en/starter/generator/command_line_tools.md

```diff
@@ -0,0 +1,154 @@
+
+-   IBM API Connect developer toolkit CLI: `apic swiftserver`.
+
+You use these command line tools to create and configure Swift Server Generator applications. The table below describes using the Yeoman command-line options. If you are familiar with the IBM API Connect developer toolkit and prefer to use that you can refer to the Swift [Server Generator installation and command line options](https://www.ibm.com/support/knowledgecenter/en/SSMNED\_5.0.0/com.ibm.apic.toolkit.doc/capic\_swift\_overview.html).
+
+The command-line tools provide an [Application generator](#application-generator) to create a new Swift Server Generator application and a number of sub-generators to define models and properties within the application, as described in the following table:
+
+*Table 1. Available commands*
+
+| Command                           | Command type                                   | Description                                                                                                                                                                                                                                                            |
+|---------------------------------  |------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| **`yo swiftserver`**              | [Application generator](#application-generator) | Create a new Swift Server Generator application.                                                                                                                                                                                                                       |
+| **`yo create swiftserver:model`** | [Model generator](#model-generator)             | Add a new model to a Swift Server Generator application.                                                                                                                                                                                                               |
+| **`yo swiftserver:property`**     | [Property generator](#property-generator)       | Add a new property to an existing model.                                                                                                                                                                                                                               |
+| **`yo swiftserver:refresh`**      | Refresh generator                              | Generate OpenAPI (Swagger 2.0) and product definition YAML files, by convention named *`application`*`.yaml` and *`application`*`-product.yaml`. If the product file already exists it is not regenerated. The OpenAPI (Swagger 2.0) definition is always regenerated. |

```

Comment:
The description here is not correct and refers to API Connect-specific functionality.
Should be:
`Generate OpenAPI (Swagger 2.0) definition YAML file, by convention named after your application (e.g. appname.yaml). This file is regenerated each time the refresh generator runs.`